src/gamma_inc.jl

```diff
@@ -715,7 +715,9 @@ function gamma_inc_inv_qsmall(a::Float64, q::Float64, qgammaxa::Float64)
         ck3 = (@horner(l, @horner(b, -12, -24, -11), @horner(b, 12, 24, 6), @horner(b, -6, -9), 2))/6.0
         ck4 = (@horner(l, @horner(b, 72, 162, 120, 25), @horner(b, -72, -168, -114, -12), @horner(b, 36, 84, 36), @horner(b, -12, -22), 3))/12.0
         x0 = x0 - l + b*r*@horner(r, ck1, ck2, ck3, ck4)
-    else
+    elseif x0 > 1
+        # The x0 > 1 condition isn't in the original version but without it
+        # the update in the branch can cause negative initial x0
```

Comment:
I wonder if adding more terms (`ck2` etc.) as in the other branch would be an alternative. And what is lost by not updating `x0`.

Reply:
I tried that a bit but it doesn't seem to help so I'm wondering if the expansion is only valid/useful when x0 > 1. Notice that the the first branch, which uses more terms, requires x0 to be even larger. It is just for choosing the starting value for the Newton iterations so I think the worst consequence could be that slightly more iterations are required.